src/Umbraco.Web.UI.Client/src/common/resources/content.resource.js

```diff
@@ -23,7 +23,7 @@
   * </pre>
   **/
 
-function contentResource($q, $http, umbDataFormatter, umbRequestHelper) {
+function contentResource($q, $http, $routeParams, umbDataFormatter, umbRequestHelper) {
```

Comment:
The service should definitely not rely on $routeParams. The `options.cultureName` is there for a reason and it needs to be explicitly passed in == no magic. If `getChildren` requires a cultureName then the options needs to specify that. I'll fix up.